hardware/include/iob_m_if.vh

```diff
@@ -0,0 +1,13 @@
+    //Master i/f (do not remove indentation)
```

Comment:
File exists

Reply:
The parameters used to configure the width are not the same